apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -269,9 +276,13 @@ export const getLegalArguments = (
   let articles = `${lawsBroken[0][1]}.`
 
   for (let i = 1; i < lawsBroken.length; i++) {
+    const noArticle = articles === '0.'
```

Comment:
We can check this before the for loop to avoid checking this for every instance, right?